pundit.gemspec

```diff
@@ -19,12 +20,4 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
 
   gem.add_dependency "activesupport", ">= 3.0.0"
-  gem.add_development_dependency "activemodel", ">= 3.0.0"
```

Comment:
Bundler's version-and-platform finding mechanism finds a gem version that suits the current running Ruby.

In order to give more background, here is an example.

The Bundler mechanism assists us when it comes to secondary deps: let's say a test tool that we explicitly refer to has a dependency. We get that pulled in implicitly. That dep needs to be "> 2" on Ruby 2+ and "< 2" on Ruby 1.9. If we're unlucky and have mentioned all this in the gemspec, we have to find and keep updated a least common denominator and that could be tricky.